<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GBSubscribeDlg</class>
 <widget class="QWidget" name="GBSubscribeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GBSubscribeDlg</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item row="0" column="3">
       <widget class="QLineEdit" name="le_ipp">
        <property name="placeholderText">
         <string>ip:port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_gbid"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>国标id:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>国标ipp:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>订阅开始时间:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateTimeEdit" name="dte_startTime">
        <property name="displayFormat">
         <string>yyyy-MM-ddTHH:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>订阅结束时间:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDateTimeEdit" name="dte_endTime">
        <property name="displayFormat">
         <string>yyyy-MM-ddTHH:mm:ss</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>报警订阅</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pb_startAlarm">
        <property name="text">
         <string>开启订阅</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_stopAlarm">
        <property name="text">
         <string>关闭订阅</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>移动设备订阅</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="pb_startDevice">
        <property name="text">
         <string>开启订阅</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_stopDevice">
        <property name="text">
         <string>关闭订阅</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>PTZ精准位置变化订阅</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="pb_startPTZ">
        <property name="text">
         <string>开启订阅</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_stopPTZ">
        <property name="text">
         <string>关闭订阅</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>目录订阅</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QPushButton" name="pb_startCatalog">
        <property name="text">
         <string>开启订阅</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_stopCatalog">
        <property name="text">
         <string>关闭订阅</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
